.gitignore

```diff
@@ -27,3 +27,4 @@ gomock_reflect_*
 /deploy/config.yaml
 **/*.swp
 /portal/node_modules/
+.idea*
```

Comment:
I am happy to ack this change, I have but a simple ask. Please make it a separate commit so it is not mixed with the code changes. Thank you.